pkg/controller/masteruserrecord/sync.go

```diff
@@ -116,6 +109,7 @@ func (s *Synchronizer) synchronizeStatus() error {
 	// Align readiness even if the user account statuses were not changed.
 	// We need to do it to cleanup outdated errors (for example if the target cluster was unavailable) if any
 	s.alignReadiness()
+	s.logger.Info("updating MUR status", "mur", s.record)
```

Comment:
Same here and in most other places. Wouldn't 
```suggestion
	s.logger.Info("updating MUR status")
```
be enough?

Reply:
see https://github.com/codeready-toolchain/host-operator/pull/165#discussion_r398366374

Reply:
done in 10097b1